### PR TITLE
help: BufCombC: change wording

### PR DESCRIPTION
**Pull request by tiagmoraismorgado**
fine tuning language details

### CODE REVIEW

**Comment by snappizz** [comment]
A better name for this commit would have been "help: BufCombC: change wording."

**Comment by tiagmoraismorgado** [comment]
thanks for helping me finding attention in detail. something extremely important in anything you might end up doing

**Comment by tiagmoraismorgado** [comment]
or thanks for the tip, otherwise

File: HelpSource/Classes/BufCombC.schelp [hunk 1]
```diff
@@ -6,12 +6,7 @@ categories::  UGens>Delays>Buffer
 
 Description::
 
-Comb delay line with cubic interpolation which uses a buffer for its
-internal memory. See also  link::Classes/BufCombN::  which uses no
-interpolation, and  link::Classes/BufCombL::  which uses linear
-interpolation. Cubic interpolation is more computationally expensive
-than linear, but more accurate.
-
+UGen consisting of a Comb delay line featuring cubic interpolation, taking advantage of a buffer for internal memory. Check link::Classes/BufCombN::  (using no interpolation), and link::Classes/BufCombL::  (uses linear interpolation). Cubic interpolation is more CPU intensive than linear, yet more accurate (sounds better).
```
Comment (vivid-synth) [inline HelpSource/Classes/BufCombC.schelp]: I personally find this wording less clear. Your note at the end ("(sounds better)") can be a fine addition, though.

Comment (tiagmoraismorgado) [inline HelpSource/Classes/BufCombC.schelp]: can you help me correcting this?

Comment (vivid-synth) [inline HelpSource/Classes/BufCombC.schelp]: @tiagmoraismorgado I'm not sure what it is that you're fixing: the language in the original is clear to me. Is there something you feel is missing or unclear?

Comment (tiagmoraismorgado) [inline HelpSource/Classes/BufCombC.schelp]: sounds better

**Comment by tiagmoraismorgado** [comment]
greetings everyone

i allowed edits in this document a couple of weeks ago. the pull request is still opened and there are requested changes. any help regarding fixing the issues related to the pull request and eventually committing it is welcome

kind regards

**Comment by snappizz** [comment]
I don't feel there is anything worth salvaging in this PR, and the author has moved on to other development tasks.

**Pull request closed without merging**